tests/test_definitions/column_map_expectations/expect_column_values_to_be_in_type_list.json

```diff
@@ -93,6 +93,15 @@
           "b": "BIT",
           "s": "VARCHAR",
           "s1": "VARCHAR"
+        },
+        "bigquery": {
+          "x": "INTEGER",
+          "y": "NUMERIC",
+          "z": "STRING",
+          "n": "STRING",
+          "b": "BOOL",
+          "s": "STRING",
+          "s1": "STRING"
```

Comment:
`expect_column_values_to_be_in_type_list` required a bigquery-specific schema and comparison that used that schema